### PR TITLE
feat(Musixmatch): support for new subdomain

### PR DESCRIPTION
**Pull request by xKubsoneQ**
## Description 
Support for new subdomain, /album presence, fixed bug in /search presence

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

</details>

### CODE REVIEW

**Comment by xKubsoneQ** [comment]
sorry for old commits, idk why-

File: websites/M/Musixmatch/presence.ts [hunk 1]
```diff
@@ -1,45 +1,114 @@
...
+		} else if (pathname.includes("/tool")) {
+			presenceData.details = "Editing lyrics";
+			presenceData.state = `${
+				document.querySelector(
+					"#__next > div > div > div > div > div > div > div > div.css-1dbjc4n.r-1p0dtai.r-1d2f490.r-12vffkv.r-u8s1d.r-zchlnj.r-ipm5af > div.css-1dbjc4n.r-13awgt0.r-12vffkv > div > div > div > div > div > div > div:nth-child(1) > div > div:nth-child(1) > div:nth-child(1) > div.css-1dbjc4n.r-13awgt0.r-1jkjb > div.css-901oao.css-bfa6kz.r-a023e6.r-rjixqe.r-fdjqy7"
+				).textContent
+			} - ${
+				document.querySelector(
+					"#__next > div > div > div > div > div > div > div > div.css-1dbjc4n.r-1p0dtai.r-1d2f490.r-12vffkv.r-u8s1d.r-zchlnj.r-ipm5af > div.css-1dbjc4n.r-13awgt0.r-12vffkv > div > div > div > div > div > div > div:nth-child(1) > div > div:nth-child(1) > div:nth-child(1) > div.css-1dbjc4n.r-13awgt0.r-1jkjb > div.css-901oao.css-bfa6kz.r-jwli3a.r-1inkyih.r-1kfrs79.r-rjixqe.r-fdjqy7"
+				).textContent
```
Comment (theusaf) [inline websites/M/Musixmatch/presence.ts:109]: These are very long selectors. I'm wondering if there are any shorter or more human-readable selectors.

**Review by theusaf — CHANGES_REQUESTED** [review]
Move `wesites/M/Musixmatch/dist/metadata.json` to `websites/M/Musixmatch/metadata.json`. `metadata.json` is no longer being stored in `dist`.

**Comment by theusaf** [comment]
> sorry for old commits, idk why-

Looking at your repository (branches `main` and `mxm-fix`), it seems that you are behind in hundreds of commits and have some additional commits from your contributions. This seems to be causing a merge conflict in this PR. 

You may need to clean up/erase the commit history of your branches and pull the latest changes.

**Pull request closed without merging**